src/main/java/com/conveyal/datatools/manager/gtfsplus/GtfsPlusValidation.java

```diff
@@ -188,7 +188,7 @@ private static void validateTable(
 
         if (tableIsDirections && !gtfsRoutes.isEmpty()) {
             // After we're done validating all the table values, check if every route was checked off in directions.txt
-            issues.add(new ValidationIssue(tableId, "route_id", -1, "Directions table does not define direction names for all routes."));
+            issues.add(new ValidationIssue(tableId, null, -1, "Directions file doesn't define directions for all routes listed in routes file"));
```

Comment:
```suggestion
            issues.add(new ValidationIssue(tableId, null, -1, "The directions file doesn't define directions for all routes listed in the routes file"));
```
or maybe say "routes listed in "routes.txt" explicitly.

Reply:
I like that better as well, I asked them what they think

Reply:
Alright we have the go-ahead for routes.txt which is my favourite